.github/workflows/test.yml

```diff
@@ -127,7 +127,7 @@ jobs:
     name: Test Suite
     runs-on: ubuntu-20.04
     if: always()
-    needs: checks
+    needs: [changes, checks]
```

Comment:
note that that I believe the only other workflow with this issue is the integration comment trigger one- the fix is less simple than this so omitting that from this PR, plus it isn't part of the merge queue, the issue only manifests as a failure to properly update the PR status check.